### PR TITLE
added a sharper image and fixed the reference

### PR DESCRIPTION
**Pull request by rubenverboon**
This PR adds a sharper image for Retina displays. Furthermore the link now goes to the 'My Tudelft' tab.

### CODE REVIEW

File: clean.js [hunk 1]
```diff
@@ -14,8 +14,10 @@ navFrame.ready(function(){
 	logo.wrap("<td id='logo'></td>").parent().insertBefore(firstMenuItem);
 
 	/* Make logo a _blank link */
-	logo.find("a").attr("target", "_blank");
-	
+	logo.find("a").attr("href", "https://blackboard.tudelft.nl/webapps/portal/frameset.jsp?tab_tab_group_id=_9_1");
+	logo.find("a").attr("target", "_top");
+	logo.find("img").attr("src", chrome.extension.getURL("tudelft_logo.png"));
```
Comment (hermanbanken) [inline clean.js]: Dit alleen doen voor Retina?

**Comment by jgadelange** [comment]
This makes this extension very TUDelft specific. Is that the intention?

**Comment by hermanbanken** [comment]
Not really. But we can easily provide a list of "profiles" later on I guess. Currently the plugin only activates at blackboard.tudelft.nl/\* anyway.

**Comment by jgadelange** [comment]
In that case: :shipit:

**Pull request merged**